builder/chroot-script.sh

```diff
@@ -204,7 +206,10 @@ apt-get install -y \
 pip install docker-compose
 
 # set up Docker APT repository and install docker-engine package
-curl -sSL https://get.docker.com | /bin/sh
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
There is a function get_gpg which has the fingerprint and an optional download url.
It will also check the downloaded key for validity which is not done by this ` curl | thing` .
Please have a look at [line98](https://github.com/hypriot/image-builder-rpi/blob/effcb7794ea745222a90ce0dfbed02d2b3787505/builder/chroot-script.sh#L98-L100) for an example.